pages/common/ugrep.md

```diff
@@ -0,0 +1,37 @@
+# ugrep
+
+> Ultra fast search tool with query UI, fuzzy search and more.
+> Try also `ugrep --help {{something}}`.
+> More information: <https://github.com/Genivia/ugrep>.
+
+- Start a [q]uery TUI to search files in the working directory recursively (CTRL-Z for help):
+
+`ugrep -Q`
```

Comment:
We prefer long arguments if available.
```suggestion
`ugrep --query`
```

Reply:
Um, all short options have long versions but I do not think it is a good idea to replace short options by long ones. The other grep-like search tool tldr pages also use short options. Please keep `-Q` in this case. Thanks.